css/disciplines-slider.styl

```diff
@@ -0,0 +1,123 @@
+DISCIPLINE_COLOR_ALL = #0072FF
+DISCIPLINE_COLOR_LANGUAGE = #9DDDF9
+DISCIPLINE_COLOR_ASTRONOMY = #512BAD
+DISCIPLINE_COLOR_NATURE = #46D178
+DISCIPLINE_COLOR_PHYSICS = #F27441
+DISCIPLINE_COLOR_CLIMATE = #A1BFBE
+DISCIPLINE_COLOR_LITERATURE = #EADF5C
+DISCIPLINE_COLOR_BIOLOGY = #18807F
+DISCIPLINE_COLOR_HUMANITARIAN = #C46FC6
+DISCIPLINE_COLOR_MEDICINE = #EF4343
+DISCIPLINE_COLOR_HISTORY = #F9DB46
+DISCIPLINE_COLOR_SOCIAL_SCIENCE = #7EA8CE
+
+.filmstrip--disciplines ul
+  display: flex;
+
+  .filmstrip--disciplines__discipline-card
+    margin: 0.28em
+    text-align: center
+    width: 5em
+    font-size:25px
+    height: 4em
+    border-radius: 1em
+    border: 0
+    padding: .5em
+    color: #006DFF
+    background-color: white
+    min-width: 100px
+    &:hover,
+    &.filmstrip--disciplines__discipline-card--active
+      color: white
+      cursor: pointer
+
+    .filmstrip--disciplines__discipline-card__icon
+      font-size: 50px
+      width: 1.5em
+      height: 2em
+      margin: 0 auto
+
+    p
+      margin-top: 0.333em
+      margin-bottom: 0px
+      font-size: 14px
+
+  .filmstrip--disciplines__discipline-card--discipline-all
+    &:hover,
+    &.filmstrip--disciplines__discipline-card--active
+      background-color: DISCIPLINE_COLOR_ALL
```

Comment:
this doesn't seem to be working quite right, it's on all the time. See https://github.com/zooniverse/Panoptes-Front-End/issues/2373 for more on this and some related issues which don't seem to match what we have in original miniztm branch